addon/components/stripe-element.js

```diff
@@ -66,7 +69,9 @@ export default Component.extend({
 
   willDestroyElement() {
     this._super(...arguments);
-    get(this, 'stripeElement').unmount();
+    const stripeElement = get(this, 'stripeElement');
+    this.stripev3.addStripeElement(stripeElement);
```

Comment:
just wondering, why add something to this.stripev3 when we are in progress of tearing everything down?

Reply:
Oop!  Merge mistake on my end.  Should be `removeStripeElement` rather than `addStripeElement`